index.js

```diff
@@ -99,11 +123,15 @@ function loadConfig(program) {
 
       if (!program.hostname && !parsedConfig.hostname) {
         if (fs.existsSync(HOSTNAME_PATH) && fs.statSync(HOSTNAME_PATH).isFile()) {
-          parsedConfig.hostname = fs.readFileSync(HOSTNAME_PATH).toString().trim().replace(HOSTNAME_IP_REGEX, '')
+          parsedConfig.hostname = fs.readFileSync(HOSTNAME_PATH)
+            .toString()
```

Comment:
You can pass encode parameter to `readFIleSync` so you don't have to run to string in process

```javascript
hostname = fs.readFIleSync(HOSTNAME_PATH, 'utf8').trim()
```